examples/LAPACK/foo.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream> // std::cout
+#include <cblas.h>
```

Comment:
I don't know. I think they are solving the same problem in slightly different ways. The creation of cblas can be turned off. I only added it to use it in the example. For suitesparse cblas is not used.

All the different blas, OpenBlas, ATLAS, LAPACK, LAPACKE are confusing me. I know suitesparse can use BLAS and LAPACK. About the others I don't know